autogen/agentchat/contrib/capabilities/context_handling.py

```diff
@@ -24,12 +24,10 @@ class TransformChatHistory:
     This capability class enables various strategies to transform chat history, such as:
     - Truncate messages: Truncate each message to first maximum number of tokens.
     - Limit number of messages: Truncate the chat history to a maximum number of (recent) messages.
-    - Limit number of tokens: Truncate the chat history to number of recent N messages that fit in
-    maximum number of tokens.
```

Comment:
I believe these were broken to avoid lines being too long. What's the reason for this change?

Reply:
You're right, the line breaks might have been intentional for readability.

My reason for removing them was to create a more compact style of the docstring. When I deleted the line breaks, I was looking at line 38, starting with "A hook is added to", which is a long line and has not been broken. 

In other class descriptions, I see lines are sometimes broken, therefore I assume there is no clear guideline for line splits. I understand the argument for line breaks and keeping them sounds good.

Reply:
I think we should leave line breaks like this. I think the general guidance is the same as for python code which we have set to 120 chars wide

Reply:
Ok, then I also think the line breaks should stay

Reply:
@jackgerrits thanks for bringing this up. I have reverted deleting the line breaks and I have added one line break to avoid this line also beeing too long